tests/integration/test_reinstall.py

```diff
@@ -28,15 +28,19 @@
 from cylc.flow.install import (
     reinstall_workflow,
 )
+from cylc.flow.option_parsers import Options
 from cylc.flow.scripts.reinstall import (
-    ReInstallOptions,
+    get_option_parser as reinstall_gop,
     reinstall_cli,
 )
+from cylc.flow.terminal import cli_function
 from cylc.flow.workflow_files import (
     WorkflowFiles,
 )
 
 
+ReInstallOptions = Options(reinstall_gop())
```

Comment:
Why move this here?

Reply:
1. It's not used anywhere else.
2. The reason `options = REINSTALL_OPTS; options.extend(OTHER_OPTS)` was being run multiple times was the presence of this line. Let us not do it if it's not necessary.

Reply:
(This object is only used by the tests so we don't need to initiate it as a global variable in the reinstall code)

Reply:
> 2\. The reason `options = REINSTALL_OPTS; options.extend(OTHER_OPTS)` was being run multiple times was the presence of this line. Let us not do it if it's not necessary.

The problem was not that `get_option_parser()` was being called multiple times, that should be able to happen without issue. The problem was `options = REINSTALL_OPTS; options.extend(OTHER_OPTS)` causing `REINSTALL_OPTS` to be mutated

Reply:
> that should be able to happen without issue

Yes, but also see point 1